tests/storage/cdi_clone/test_clone.py

```diff
@@ -114,8 +114,8 @@ def test_successful_clone_of_large_image(
         pytest.param(
             {
                 "dv_name": "dv-source",
-                "image": f"{Images.Cirros.DIR}/{Images.Cirros.QCOW2_IMG}",
-                "dv_size": Images.Cirros.DEFAULT_DV_SIZE,
+                "image": FEDORA_LATEST["image_path"],
```

Comment:
```suggestion
                "image": FEDORA_LATEST.get("image_path"),
```
In cases where we do not have a fedora image ready for a new arch
this is needed for the collection part (the test will not be executed if the image is missing as there will be no arch marker on it)

Applies to all places where accessing such keys in pytest.param